### ISSUE TITLE
TypeError: sqlweb.parseSql is not a function

### ISSUE THREAD

**Issue by Lars-** [ujjwalguptaofficial/idbstudio#13]
When running `idbstudio --sql "select * from customers"` I get the error `TypeError: sqlweb.parseSql is not a function`
Is it possible to fix this?

**Comment by NotDemin** [comment]
did you fix it ?

**Comment by Lars-** [comment]
Nope. Ended up using something else I think, but I can't remember exactly anymore.
